internal/backend/auth/authloginhttpsvc/google.go

```diff
@@ -0,0 +1,57 @@
+package authloginhttpsvc
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/dghubble/gologin/v2/google"
+
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+
+	"golang.org/x/oauth2"
+	googleOAuth2 "golang.org/x/oauth2/google"
+)
+
+func registerGoogleOAuthRoutes(mux *http.ServeMux, cfg oauth2Config, onSuccess func(sdktypes.User) http.Handler) error {
+	if cfg.ClientID == "" {
+		return errors.New("google login is enabled, but missing GOOGLE_CLIENT_ID")
```

Comment:
We need to make sure this env variables are not mixed the with google integration env variables (which might have the same values, but should have different names)

Reply:
ticket ?

Reply:
https://linear.app/autokitteh/issue/ENG-481/use-config-instead-of-direct-env-vars-in-integrations